Resource/PYTHON/multilateration/multilateration.py

```diff
@@ -0,0 +1,287 @@
+import math
+
+import numpy as np
+
+import multilateration.parameters as param
+
+# Initializing the variables maximum_time_diff and max_hydrophone_distance.
+# These values are updated in the function initialize_trilateration_globals
+
+# Initialized to -1 such that it's easy to check if the variables are incorrect
+
+max_hydrophone_distance = -1.0
+max_time_diff = -1.0
```

Comment:
If you want a good value to initialize this global to then use `None` instead of `-1.0`. Then you can use
``` python
if max_time_diff is None:
    pass
```